plugins/module_utils/network/eos/config/ospfv3/ospfv3.py

```diff
@@ -147,6 +146,26 @@ def _compare(self, want, have):
 
     def _global_compare(self, want, have):
         for name, entry in iteritems(want):
+            if name == "timers":
+                if entry.get("throttle"):
+                    self._module.warn(
+                        "'timers throttle' command is deprecated by 'timers lsa / timers spf'"
+                    )
+                    entry.pop("throttle")
+                if entry.get("lsa") and not isinstance(entry["lsa"], dict):
+                    modified = {
+                        "timers": {
+                            "lsa": {"direction": "rx", "min": entry["lsa"]}
+                        }
+                    }
+                    self._module.warn(
+                        " ** 'timers lsa arrival' has changed to 'timers lsa rx min interval' from eos 4.23 onwards. ** "
+                        " \n** Your task has been modified to use {0}. ** "
+                        " \n** timers.lsa of type int will be deprecated by '2023-08-31' ** ".format(
```

Comment:
Two nitpicks here:
* The option will already be deprecated by this PR, it will be _removed_ in two years
* The theory is not to promise any removals to happen in exactly two years, but to schedule them all together for a release after the next January or July following the two year deprecation period, whichever is sooner. This is so that all the removals happen together and we don't get a hodgepodge of major versions coming out at odd times.

Thus for this change, it's right after July 2023, so the next window is sometime after January 2024
```suggestion
                        " \n** timers.lsa of type int will be removed after '2024-01-01' ** ".format(
```